things/api.py

```diff
@@ -510,7 +534,7 @@ def upcoming(**kwargs):
 
     For details on parameters, see `things.api.tasks`.
     """
-    return tasks(start_date=True, start="Someday", **kwargs)
+    return tasks(start_date="future", start="Someday", **kwargs)
```

Comment:
Why? Does `start_date=True` not suffice?

Reply:
This would show "yellow" tasks from the Today list in the upcoming list.

Reply:
Actually I noticed that the `Upcoming` view in the app also shows tasks with deadlines in the future. This is not reflected in the API currently.

Reply:
👍